design/vela-core/environment.md

```diff
@@ -1,294 +1,262 @@
 
 ## Background
 
-A team of application development usually needs to initialize some shared environments for developers to deploy applications to.
+A application development team usually needs to initialize some shared environments for developers to deploy applications to.
 For example, a team would initialize two environments, a *dev* environment for testing applications,
 and a *prod* environment for running applications to serve live traffic.
-The environment is a logical concept grouping common resources that multiple application deployments would depend on.
 
-A list of resources we want to initialize per environment:
+The environment is a logical concept grouping common resources that many applications would depend on. Below are a list of cases:
 
-- K8s Clusters. Different environments might have clusters of different size and version, e.g.
-  small v1.10 k8s cluster for *dev* environment and large v1.9 k8s cluster for *prod* environment.
-- Admin policies. Production environment would usually set global policies on all application deployments,
+- Kubernetes clusters. These include not only existing clusters but also new clusters to create during environment setup.
+- Admin policies. Production environment would usually set global policies for all application deployments,
   like chaos testing, SLO requirements, security scanning, misconfiguration detection.
-- Operators & CRDs. Environments contains a variety of Operators & CRDs as system capabilities.
-  These operators can include domain registration, routing, logging, monitoring, autoscaling, etc.
+- System components. That include installing system level Operators & CRDS like FluxCD, KEDA, OpenKruise,
+  shared Namespaces, Observability (Prometheus, Grafana, Loki), Terraform Controller, KubeFlow Controller, etc.
 - Shared services. Environments contains a variety of system services that are be shared by applications.
   These resources can include DB, cache, load balancers, API Gateways, etc.
 
-With the introduction of environment, the user workflow on Vela works like:
+In this doc, we will provide solutions and case studies on how to use KubeVela to make environments, initialize shared resources,
+and compose application rollout across multiple environments.
 
-- Platform team defines system Components for Environments.
-- Platform team initializes Environments using system Components.
-- Platform team defines user Components for Applications.
-- Developers choose user Components and Environments to deploy Applications.
 
-When initializing environments, there are additional requirements:
+## Using KubeVela to initialize Environment
 
-- The ability to describe dependency relations between environments. For example, both *dev* and *prod* environments
-  would depend on a common environment that sets up operators both *dev* and *prod* would use.
+An environment is just a logical concept that groups a bundle of resources for multiple applications to share.
```

Comment:
This meaning has been expressed in a line above https://github.com/oam-dev/kubevela/pull/2199/files#diff-b42a63b4159dee9cf6945f592072aa04d2fb83b18a52c85ef3339f9df1f1bf24R9.

Reply:
Well. This sentence is good enough as the beginning of the section.